test/FunctionalTests/Tests/DataOData/Tests/OData.PluggableFormat.Tests/ServiceProviderBuilderHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿//---------------------------------------------------------------------

```

Comment:
Remove this file

Reply:
This file was renamed to `ServiceProviderHelper`. Replace `ServiceProviderBuilderHelper` with `ServiceProviderHelper` in https://github.com/OData/odata.net/pull/2910/files#diff-05c362e15358d976c1f3cd9090f75c6e726b63be9542ee4dad138782583f2015L111 
 - test/FunctionalTests/Tests/DataOData/Tests/OData.PluggableFormat.Tests/Avro/Test/ODataAvroScenarioTests.cs around line 112 (in case the link doesn't work)